src/dataFiles.ts

```diff
@@ -1,16 +1,30 @@
 import { readdirSync, statSync } from "fs";
-import { extname, join } from "path";
-import { DATA_EXTENSIONS } from "./constants.js";
+import { basename, extname, join } from "path";
+import {
+	BADFOLDERS_REGEX,
+	IGNORED_FOLDERS,
+	VIDEO_EXTENSIONS,
+} from "./constants.js";
 import { getRuntimeConfig } from "./runtimeConfig.js";
 
+function checkIgnorePath(root: string, isDir: boolean) {
```

Comment:
can we find a name for `checkIgnorePath` that makes it clearer what it does without having to look at its implementation? here's my best idea but open to others 

```suggestion
function shouldIgnorePathHeuristically(path: string, isDir: boolean) {
```